### DISCUSSION TITLE
Configuration and selection of multiple different self built server lists

### DISCUSSION THREAD

**Discussion by itbencn** [rustdesk/rustdesk#4911] [General]
When there are two or more self built server nodes, client configuration is cumbersome, and only one configuration information can be saved at the same time.

It is recommended to add a server node selection function in the status bar at the bottom of the homepage, as well as better support for saving node configurations in the form of local files, to facilitate migration

**Comment by itbencn** [comment]
Another challenging idea:
Under the premise of supporting multiple nodes, record the information of that node in each session record and automatically use the corresponding node when connecting to the session in the future

**Comment by itbencn** [comment]
@rustdesk

**Comment by cloudmaker97** [comment]
This is an absolutely amazing idea! I have to switch the servers more than twice a day. This would improving the QOL

**Comment by rustdesk** [comment]
close since duplicate to https://github.com/rustdesk/rustdesk/discussions/1258
